br.com.wiselabs.simplexity.yml

```diff
@@ -0,0 +1,56 @@
+app-id: br.com.wiselabs.simplexity
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node20
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=home
+build-options:
+  append-path: /usr/lib/sdk/node20/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: simplexity
+    buildsystem: simple
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/simplexity/flatpak-node/cache
+        npm_config_cache: /run/build/simplexity/flatpak-node/npm-cache
+        npm_config_offline: 'true'
+    build-commands:
+      - yarn install --offline
+      - |
+        . ../flatpak-node/electron-builder-arch-args.sh
+        yarn run dist -- $ELECTRON_BUILDER_ARCH_ARGS  --linux --dir
+      - cp -a release-builds/linux*unpacked /app/main
+      - install -Dm755 -t /app/bin/ ../run.sh
+      - install -Dm644 img/icon.svg /app/share/icons/hicolor/scalable/apps/br.com.wiselabs.simplexity.svg
+    subdir: main
+    sources:
+      - type: archive
+        url: https://github.com/Wiselabs/simplexity/releases/download/v1.0.1/simplexity-1.0.1-src-with_node_modules.tar.gz
+        sha256: 2ba110f4edf187960b7cd3b9a6f0094e681c19c4e36f23475bdcca146a785a11
+        dest: main
+        strip-components: 0
+      - type: file
+        path: br.com.wiselabs.simplexity.appdata.xml
+      - type: file
+        path: br.com.wiselabs.simplexity.desktop
+      - generated-sources.json
+      # Wrapper to launch the app
+      - type: script
+        dest-filename: run.sh
+        commands:
+          - install -Dm644 br.com.wiselabs.simplexity.appdata.xml /app/share/applications/br.com.wiselabs.simplexity.appdata.xml
+          - install -Dm644 br.com.wiselabs.simplexity.desktop /app/share/applications/br.com.wiselabs.simplexity.desktop
```

Comment:
these two should be inside post-install or build-commands. once installed it is immutable.